manifests/storage_migration_crd.yaml

```diff
@@ -10,86 +12,85 @@ spec:
     plural: storageversionmigrations
     singular: storageversionmigration
   scope: Cluster
```

Comment:
suggest also explicitly setting `preserveUnknownFields: false` to ensure an existing persisted CRD migrated from v1beta1 turns on pruning